packages/host/types/ember-keyboard/modifiers/on-key.d.ts

```diff
@@ -0,0 +1,15 @@
+import Modifier from 'ember-modifier';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    Positional: [keyCombo: string, callback?: (ev: KeyboardEvent) => void];
+    Named: {
+      event?: string;
+      activated?: boolean;
+      priority?: number;
+    };
+  };
+}
+
+export default class OnKeyModifier extends Modifier<Signature> {}
```

Comment:
If we produce enough of these maybe we can just [do this](https://github.com/adopted-ember-addons/ember-keyboard/issues/464#issuecomment-1709156176) 😆